versioned_docs/version-23.10/integrations/data-analytics/sc-canopsis-events.md

```diff
@@ -0,0 +1,422 @@
+> For others, their default values defined in the connector are canopsis_port:443 and canopsis_pbehavior-reasons:/api/v4/pbehavior-reasons
+
+3. Check if command returns a data structure of this form:
+```json
+   "data":[
+      {
+         "_id":"XXXX",
+         "name":"NAME",
+         "description":"DESCRIPTION"
+      }
+   ]
+```
+
+#### Request pbehavior-types route
+1. Log in to the server that you configured to send events to Canopsis (your central 
+server, a remote server or a poller).
+2. Run the following command:
+
+```shell
+curl -X GET -H 'accept: application/json' -H 'x-canopsis-authkey: <canopsis-auth-token>' '<https_canopsis_host_url>:<canopsis_port><canopsis_pbehavior-types>'
+```
+
+> Replace all the *`<xxxx>`* inside the above command with the correct value for authentification and Canopsis host. 
+> For others, their default values defined in the connector are canopsis_port:443 and canopsis_pbehavior-types:/api/v4/pbehavior-types
+
+3. Check if command returns a data structure of this form:
+```json
+"data":[
+      {
+         "_id":"ec35c069-1651-4ee1-8944-3e5574e7b516",
+         "name":"Default active",
+         "description":"Default active",
+         "type":"active",
+         "priority":2,
+         "icon_name":"",
+         "color":"#2FAB63"
+      },
+      {
+         "_id":"470c469c-77bc-402c-910f-30a8b2584343",
+         "name":"Default inactive",
+         "description":"Default inactive",
+         "type":"inactive",
+         "priority":1,
+         "icon_name":"brightness_3",
+         "color":"#979797"
+      },
+      {
+         "_id":"5ea9d2d8-0f16-4e19-bcca-64b1e96e00fa",
+         "name":"Default maintenance",
+         "description":"Default maintenance",
+         "type":"maintenance",
+         "priority":3,
+         "icon_name":"build",
+         "color":"#BF360C"
+      },
+      {
+         "_id":"1fb65097-ddaa-4e99-9239-8263095c156c",
+         "name":"Default pause",
+         "description":"Default pause",
+         "type":"pause",
+         "priority":4,
+         "icon_name":"pause",
+         "color":"#5A6D80"
+      }
+   ]
+```
+> With default values of the Canopsis connector, it classifies the downtimes in Default maintenance type.
+
+#### Request app-info route
+1. Log in to the server that you configured to send events to Canopsis (your central 
+server, a remote server or a poller).
+2. Run the following command:
+
+```shell
+curl -X GET -H 'accept: application/json' -H 'x-canopsis-authkey: <canopsis-auth-token>' '<https_canopsis_host_url>:<canopsis_port>/api/v4/app-info'
+```
+
+> Replace all the *`<xxxx>`* inside the above command with the correct value for authentification and Canopsis host. 
+> For canopsis port the default value defined in the connector is canopsis_port:443
+
+3. This command only allows access to the Canopsis version so no need to inspect all the returned content of it.
```

Comment:
```suggestion
This command only returns the version of Canopsis.
```

Reply:
La commande retourne pleins d'autres champs donc est-ce qu'une formilation comme ça va mieux ?
`This command is only use to returns the version of Canopsis even if contains a lot of other things.`